src/VisualStudio/CSharp/Impl/CodeModel/CSharpCodeModelService.cs

```diff
@@ -1140,7 +1140,8 @@ private EnvDTE.vsCMAccess GetDefaultAccessibility(SyntaxNode node)
                 }
             }
 
-            if (node is AccessorDeclarationSyntax)
+            if (node is AccessorDeclarationSyntax || 
+                node is ArrowExpressionClauseSyntax)
             {
                 return GetAccess(node.FirstAncestorOrSelf<BasePropertyDeclarationSyntax>());
```

Comment:
Isn't this already a bug for normal accessors in the case where the accessor is more restrictive:

``` C#
     public void Foo
    {
        get { return 0; }
        $$private set { }
    }
```